app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -69,13 +71,45 @@ class BeneficiaryLiveFeed extends React.Component {
                               </UserTime>
                             );
 
-                            if (transfer.transfer_type === 'PAYMENT') {
+                            let currency, fromExchangeTransfer, recipientCurrency;
```

Comment:
it's a bit more tedious, but best practice is to make each variable a new line:

```
let currency;
let fromExchangeTransfer;
let recipientCurrency;
```

this way if anything ever changes, you'll quickly see if one should be a `const` instead, and more importantly, if one of them is ever moved to a new line, you won't accidentally end up with a global. #justjavascriptthings

Reply:
Ok, thanks for the tip!